rpi_csdt_community/settings.py

```diff
@@ -369,10 +369,10 @@
 
     # Make sure a GOOGLE_API_KEY is defined
     try:
-        GOOGLE_API_KEY  # noqa: F405
+        GOOGLE_API_KEY = 'fbdthsjhgfdhnmdjsuyr3tythdf'  # noqa: F405
```

Comment:
1. You should always use fake names that are clearly fake, like '123456789' or 'this is not a real key'
2. You don't need name error if you have defined them here.